ci/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"github.com/mysteriumnetwork/go-openvpn/ci/commands"
+	"os"
+	"strings"
+)
+
+func main() {
+	help := `
+Usage: 
+	go run ci/main.go <command>
+
+The commands are:
+	test      Runs all unit tests in project
+	help      Shows a list of commands
+`
+
+	args := os.Args[1:]
+	if len(args) < 1 {
+		fmt.Print(help)
+		commands.ExitWithError("Need subcommand as first argument")
+	}
+
+	command := strings.ToLower(args[0])
+	switch command {
```

Comment:
this works, but won't work well looking at the future. If we add tons of scripts here, we'll need to have separate helps for them and what not. The switch will be hard to read if we add them here. At least use an urfave/cli here, or better yet - something like [magefile](https://magefile.org/).